java/bench/pom.xml

```diff
@@ -400,7 +400,7 @@
       <dependency>
         <groupId>org.codehaus.janino</groupId>
         <artifactId>janino</artifactId>
-        <version>3.0.8</version>
+        <version>3.1.6</version>
```

Comment:
There was some bugs on the latest one and Spark is using 3.0.16. This should be matched with Spark.
```
    <janino.version>3.0.16</janino.version>
```